spec/pagseguro/extensions/collection_object_spec.rb

```diff
@@ -0,0 +1,82 @@
+require 'spec_helper'
+
+describe PagSeguro::Extensions::CollectionObject do
+  before(:all) do
+    # Collection
+    class MyItems
+      include PagSeguro::Extensions::CollectionObject
+    end
+
+    # Item
+    class MyItem
+      attr_accessor :name
+
+      def ==(other)
+        name.eql?(other.name)
+      end
+    end
+  end
+
+  let(:collection) do
+    MyItems.new
+  end
+
+  let(:object) do
+    item = MyItem.new
+    item.name = 'Item1'
+    item
```

Comment:
Because this way I have to ensure that I'll implement the mass assignment extension too and this is not my goal.